locales/en/messages.json

```diff
@@ -92,6 +92,10 @@
     "analyticsOptOut": {
         "message": "Opt out of the anonymised collection of statistics data"
     },
+    "connectionTimeout": {
+        "message": "Set connection timeout to allow longer initialisation on device plugin",
```

Comment:
Is this used only for autoconnect? Or when save and reboot is used too? If it's only for auto connect, I think we must use the word here. If it is for both, then the message is good.

Reply:
Yes good call, I think addressing save and reboot with this setting could be done too and could help tackle more connection issues.